modules/core/src/test/java/org/apache/ignite/internal/client/thin/CacheEntryListenersTest.java

```diff
@@ -771,6 +797,15 @@ public void assertNextCacheEvent(EventType expType, K expKey, V expVal) throws E
             assertEquals(expVal, evt.getValue());
         }
 
+        /** */
+        public void assertNextCacheEventDataNotReceived(EventType expType, String name) throws Exception {
```

Comment:
Confusing method name. And looks like we check some internal state instead of external behaviour. Also, logic related to class field names and reflection is fragile. Can we implement something like:
```
        public void assertNextCacheEvent(EventType expType, Consumer<CacheEntryEvent<? extends K, ? extends V>> checker) throws Exception {
            CacheEntryEvent<? extends K, ? extends V> evt = poll();

            assertEquals(expType, evt.getEventType());

            checker.accept(evt);
        }
```
And use it:
```
            lsnr.assertNextCacheEvent(EventType.REMOVED, evt -> assertSame(evt.getValue(), evt.getOldValue()));
```
?